### PR TITLE
TINY-4783: Fixing some issues raised by LGTM analysis tool

### PR DESCRIPTION
**Pull request by tinydylan**

### CODE REVIEW

File: modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts [hunk 1]
```diff
@@ -408,18 +408,14 @@ function DOMUtils(doc: Document, settings: Partial<DOMUtilsSettings> = {}): DOMU
     }
 
     // Make a HTML5 safe shallow copy
-    if (!deep) {
-      const clone = doc.createElement(node.nodeName);
+    const clone = doc.createElement(node.nodeName);
```
Comment (TheSpyder) [inline modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts]: could we nest this inside an `else` instead of flattening it 🤔

Comment (tinydylan) [reply modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts]: Done.

File: modules/katamari/src/test/ts/atomic/api/fun/ThunkTest.ts [hunk 2]
```diff
@@ -3,19 +3,17 @@ import fc from 'fast-check';
 import { UnitTest, Assert } from '@ephox/bedrock-client';
 
 UnitTest.test('ThunkTest', function () {
-  const testSanity = function () {
```
Comment (lnewson) [inline modules/katamari/src/test/ts/atomic/api/fun/ThunkTest.ts]: So I assume this was never running? 😂

Comment (tinydylan) [reply modules/katamari/src/test/ts/atomic/api/fun/ThunkTest.ts]: IKR?

**Pull request merged**